internal/deviations/deviations.go

```diff
@@ -72,20 +74,60 @@ import (
 	"github.com/openconfig/ondatra"
 )
 
+func matchRegex(regex string, deviceInfo string) bool {
+	match, err := regexp.MatchString(regex, deviceInfo)
+	if err != nil {
+		log.Errorf("Error with regex match %v", err)
+	} else if match {
```

Comment:
drop the "else if" and just have an un-guarded "return match"

Reply:
done.